joomla/joomla-cms/CVE-2018-6376.yaml

```diff
@@ -0,0 +1,8 @@
+title: SQLi vulnerability in Hathor postinstall message
+link: https://developer.joomla.org/security-centre/722-20180105-core-sqli-vulnerability.html
+cve: CVE-2018-6376
+branches:
+    master:
+        time:     2018-1-30
+        versions: ['>=3.7.0', '<3.8.3']
+reference: composer://joomla/joomla-cms
```

Comment:
`joomla/joomla-cms` does not seem to be a package available on packagist. how is it distributed?

Reply:
`joomla/joomla-cms` is indeed not registered on Packagist, as it is a project using Composer for its own deps, not something you can install as a dependency.